roles/sat6repro/tasks/main.yml

```diff
@@ -71,20 +82,11 @@
   command: katello-installer
   when: satelliteversion == 6.1
 - name: run Satellite 6.2 installer
-  command: satellite-installer --scenario satellite
+  command: satellite-installer --scenario satellite --foreman-proxy-dns false --foreman-proxy-dhcp false
```

Comment:
having these on resulted in an error:

```[ERROR 2016-12-13 16:17:36 main]  Could not get the ip address from fact ipaddress_virbr1 at /usr/share/foreman-installer/modules/foreman_proxy/manifests/proxydhcp.pp:8 on node sat-r220-09.lab.eng.rdu2.redhat.com```

If the user wants to test provisioning, they are going to have to turn these back on